tests/ui/associated-type-bounds/bad-bounds-on-assoc-in-trait.rs

```diff
@@ -1,5 +1,4 @@
-// NOTE: rustc cannot currently handle bounds of the form `for<'a> <Foo as Bar<'a>>::Assoc: Baz`.
-// This should hopefully be fixed with Chalk.
+// check-pass
```

Comment:
This test was already not even well-formed. I removed the items that were actually missing, and fixed the test to actually make sense.